README.md

```diff
@@ -56,8 +58,8 @@ To see the rules that this config uses, please read the [config itself](./index.
 
 @media (orientation: portrait), projection and (color) {
   .selector-i + .selector-ii {
-    background: color(rgb(0, 0, 0) lightness(50%));
-    font-family: helvetica, "arial black", sans-serif;
+    background: hsl(20deg 25% 33%);
```

Comment:
We can remove this old `color` function. 

Incidentally, the upcoming [relative color syntax](https://www.w3.org/TR/css-color-5/#relative-colors) looks great!